src/main/java/uk/gov/ons/ctp/response/collection/exercise/endpoint/CollectionExerciseEndpoint.java

```diff
@@ -689,7 +700,9 @@ private CollectionExerciseDTO getCollectionExerciseDTO(
   @RequestMapping(value = "/{id}/events", method = RequestMethod.POST)
   public ResponseEntity<?> createCollectionExerciseEvent(
       @PathVariable("id") final UUID id, final @RequestBody EventDTO eventDto) throws CTPException {
-    log.info("Creating event {} for collection exercise {}", eventDto.getTag(), id);
+    log.with("collection_exercise_id", id)
+        .with("EventTag", eventDto.getTag())
+        .info("Creating event for collection exercise");
```

Comment:
I hate these damn infos which should really be debugs - I know you didn't create this originally though